src/index.js

```diff
@@ -1,42 +1,40 @@
-/**************************************************************************
- *  (C) Copyright Mojaloop Foundation 2020 - All rights reserved.         *
- *                                                                        *
- *  This file is made available under the terms of the license agreement  *
- *  specified in the corresponding source code repository.                *
- *                                                                        *
- *  Original Author:                                                      *
- *       Rajiv Mothilal - rajiv.mothilal@modusbox.com                     *
- **************************************************************************/
 'use strict'
 
-const Server = require('./server')
-const PJson = require('../package.json')
-const { Command } = require('commander')
-const Config = require('./lib/config')
-const argv = require('./lib/argv').getArgs()
+const util = require('util')
+const Logger = require('@mojaloop/central-services-logger')
+const Config = require('./config')
+const { createServer } = require('./server')
+const { OracleDatabase } = require('./db')
 
-const Program = new Command()
+const initialize = async () => {
+  // Set log level to info
+  // eslint-disable-next-line
```

Comment:
is this linting disable needed?

Reply:
for now yes to ignore an "Arrow function should not return assignment". this was code I took from the pathfinder repo, so I didn't want to change it up here.